lib/profile/header.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
+ 
+class ProfileHeader extends StatelessWidget {  
+  final String fullName, emailAddress;
+  ProfileHeader({Key key, @required this.fullName, @required this.emailAddress}) : super(key: key);
+  
+  @override
+    Widget build(BuildContext context) {
```

Comment:
Same thing here. 

Nit-picky but typically you don't have to indent here. You can just do: 
```dart
@override
Widget build(BuildContext content) 
```